site2/docs/administration-pulsar-manager.md

```diff
@@ -21,89 +22,42 @@ docker run -it \
+If you have a large amount of data, you can use a custom database. 
 
-1. Initialize database and table structures using the [file](https://github.com/apache/pulsar-manager/tree/master/src/main/resources/META-INF/sql/postgresql-schema.sql).
+The following is an example of PostgreSQL.
 
-2. Modify the [configuration file](https://github.com/apache/pulsar-manager/blob/master/src/main/resources/application.properties) and add PostgreSQL configuration.
-
-```
+  1. Initialize database and table structures using the [file](https://github.com/apache/pulsar-manager/blob/master/src/main/resources/META-INF/sql/postgresql-schema.sql).
+  2. Download and modify the [configuration file](https://github.com/apache/pulsar-manager/blob/master/src/main/resources/application.properties), then add the PostgreSQL configuration.
+```properties
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.url=jdbc:postgresql://127.0.0.1:5432/pulsar_manager
 spring.datasource.username=postgres
 spring.datasource.password=postgres
 ```
 
-3. Compile to generate a new executable jar package.
-
-```
-./gradlew build -x test
-```
-
-### Enable JWT authentication
-
-If you want to turn on JWT authentication, configure the following parameters:
-
-* `backend.jwt.token`: token for the superuser. You need to configure this parameter during cluster initialization.
-* `jwt.broker.token.mode`: multiple modes of generating token, including PUBLIC, PRIVATE, and SECRET.
-* `jwt.broker.public.key`: configure this option if you use the PUBLIC mode.
-* `jwt.broker.private.key`: configure this option if you use the PRIVATE mode.
-* `jwt.broker.secret.key`: configure this option if you use the SECRET mode.
-
-For more information, see [Token Authentication Admin of Pulsar](http://pulsar.apache.org/docs/en/security-token-admin/).
-
-
-If you want to enable JWT authentication, use one of the following methods.
-
-
-* Method 1: use command-line tool
-
-```
-wget https://dist.apache.org/repos/dist/release/pulsar/pulsar-manager/apache-pulsar-manager-0.2.0/apache-pulsar-manager-0.2.0-bin.tar.gz
-tar -zxvf apache-pulsar-manager-0.2.0-bin.tar.gz
-cd pulsar-manager
-tar -zxvf pulsar-manager.tar
-cd pulsar-manager
-cp -r ../dist ui
-./bin/pulsar-manager --redirect.host=http://localhost --redirect.port=9527 insert.stats.interval=600000 --backend.jwt.token=token --jwt.broker.token.mode=PRIVATE --jwt.broker.private.key=file:///path/broker-private.key --jwt.broker.public.key=file:///path/broker-public.key 
+  3. Add a configuration mount and start with a docker image.
+```bash
+docker pull apachepulsar/pulsar-manager:v0.2.0
+docker run -it \
+    -p 9527:9527 -p 7750:7750 \
+    -v /your-path/application.properties:/pulsar-manager/pulsar-
+manager/application.properties
+    -e SPRING_CONFIGURATION_FILE=/pulsar-manager/pulsar-manager/application.properties \
+    apachepulsar/pulsar-manager:v0.2.0
```

Comment:
```
docker run -it \
    -p 9527:9527 -p 7750:7750 \
    -v /your-path/application.properties:/pulsar-manager/pulsar-manager/application.properties \
    -e SPRING_CONFIGURATION_FILE=/pulsar-manager/pulsar-manager/application.properties \
    apachepulsar/pulsar-manager:v0.2.0
```
Should this command look like this?

Reply:
That's it. I've simplified the Quick Start command, if you need to set environment variables, you can refer to the detailed parameters below.